### ISSUE TITLE
username/teamname rendered incorrectly

### ISSUE THREAD

**Issue by stub42** [canonical/jujucharms.com#348]
Usernames and teamnames are being rendered as the launchpad id, which is incorrect. The displayname from Launchpad should be used.

For example, 'postgresql by postgresql-charmers' should be rendered as 'postgresql by PostgreSQL Charm Maintainers'.

Using the ids creates a poor UX, as the only way of choosing the rendering is by setting it in the Launchpad team id, which is not always available and will collide with upstream maintainer teams, deb package maintenance teams and snap packaging teams.

**Comment by bac** [comment]
@stub42 I understand you have a preference for seeing the displayname and not the Launchpad ID. I don't, however, follow your point in the last paragraph.  Can you rephrase it? Since the Launchpad ID is unique (is the Launchpad displayname guaranteed to be unique?) I don't understand the collision you mention.

Thanks!

**Comment by stub42** [comment]
The Launchpad teams maintaining the charm, maintaining the snap, maintaining the deb package, maintaining the upstream production could all be distinct groups of people and require unique team names.

Current policy is to upload charms under a cs:~foo-team namespace to provide a better charmstore UX (as opposed to cs:~foo-charmers or cs:~foo-hack0rz). From the Launchpad side of things, this is rather rude as it looks like you are claiming to be upstream. And if foo-team is already claimed by the snap packagers or upstream, then you are SOL.
